services/authorize/client.go

```diff
@@ -207,6 +205,20 @@ func (c *client) DialUsingCredentialsWithContext(ctx context.Context, sess *sess
 	return err
 }
 
+func reconnectRetryInterceptor(newClientConn reconnect.NewConnectionFunc) grpc.DialOption {
+	retryIC := grpc_retry.UnaryClientInterceptor(
+		grpc_retry.WithBackoff(grpc_retry.BackoffLinear(100*time.Millisecond)),
```

Comment:
I think this should be configurable. Even which type of backoff is used.

Reply:
That can be a separate ticket for improvements